src/ai/backend/manager/repositories/artifact/repository.py

```diff
@@ -697,6 +701,34 @@ async def update_artifact_revision_bytesize(
             await db_sess.execute(stmt)
             return artifact_revision_id
 
+    @repository_decorator()
+    async def update_artifact_revision_readme(
+        self, artifact_revision_id: uuid.UUID, readme: str
+    ) -> uuid.UUID:
+        async with self._db.begin_session() as db_sess:
+            stmt = (
+                sa.update(ArtifactRevisionRow)
+                .where(ArtifactRevisionRow.id == artifact_revision_id)
+                .values(readme=readme)
+            )
+            await db_sess.execute(stmt)
+            return artifact_revision_id
+
+    @repository_decorator()
+    async def get_artifact_revision_readme(self, artifact_revision_id: uuid.UUID) -> Optional[str]:
+        async with self._db.begin_session() as db_sess:
+            result = await db_sess.execute(
+                sa.select(ArtifactRevisionRow.readme).where(
+                    ArtifactRevisionRow.id == artifact_revision_id
+                )
+            )
+            readme = result.scalar_one_or_none()
+            if readme is None:
+                raise ArtifactRevisionNotFoundError(
+                    f"Artifact revision with ID {artifact_revision_id} not found"
+                )
+            return readme
```

Comment:
The return value doesn't seem like it's Optional, but is it correct that it returns an Optional?